manual/manual/tutorials/objectexamples.etex

```diff
@@ -1091,11 +1091,15 @@ class virtual comparable =
   end;;
 \end{caml_example}
 We then define a subclass "money" of "comparable". The class "money"
-simply wraps floats as comparable objects. We will extend it below with
-more operations. We have to use a type constraint on the class parameter "x"
-because the primitive "<=" is a polymorphic function in
-OCaml.  The "inherit" clause ensures that the type of objects
-of this class is an instance of "#comparable".
+simply wraps floats as comparable objects.  Note: in real applications,
+unless you have a good reason, you should never use floats to represent
```

Comment:
This  `never, unless you have a good reason` sounds quite ambiguous. A more factual description may work better. Something like:
```latex
\footnote{Float numbers are an approximation of real numbers, they do not fit the
requirements of most real financial applications}
```
I think that is one of the rare cases where a footnote is warranted, because I do not expect readers to use this tutorial as a starting point for their law-abiding financial application.